templates/tfengine/components/resources/healthcare_datasets/main.tf

```diff
@@ -66,6 +66,23 @@ module "{{resourceName . "name"}}" {
       }
       {{end -}}
 
+      {{if has . "stream_configs" -}}
+      stream_configs = [
+        {{range $k, $v := .stream_configs -}}
+        {
+          bigquery_destination = {
+            dataset_uri = {{hcl $v.bigquery_destination.dataset_uri}}
```

Comment:
you could probably just do "{{$v.biguqery_destination.dataset_uri}}"

Reply:
Done